app/components/submission/ManuscriptMetadata/ManuscriptMetadataSchema.js

```diff
@@ -9,29 +9,23 @@ const schema = yup.object().shape({
     .max(2, `No more than 2 subject areas`)
     .required('Subject area(s) required'),
   submissionMeta: yup.object().shape({
-    discussedPreviously: yup.bool(),
-    discussion: yup.string().when('discussedPreviously', {
-      is: true,
-      then: yup.string().required('Please describe your previous discussion'),
-    }),
-    consideredPreviously: yup.bool(),
-    previousArticle: yup.string().when('consideredPreviously', {
-      is: true,
-      then: yup.string().required('Article title/reference no. is required'),
-    }),
-    cosubmission: yup.bool(),
-    cosubmissionTitle: yup
+    discussion: yup
       .string()
-      .test('namehere', 'Article title/reference no. is required', function() {
-        const { cosubmission, cosubmissionTitle, cosubmissionId } = this.parent
-        return !cosubmission || cosubmissionTitle || cosubmissionId
-      }),
-    cosubmissionId: yup
+      .notOneOf([''], 'Please describe your previous discussion')
+      .nullable(),
+    previousArticle: yup
       .string()
-      .test('namehere', 'Article title/reference no. is required', function() {
-        const { cosubmission, cosubmissionTitle, cosubmissionId } = this.parent
-        return !cosubmission || cosubmissionTitle || cosubmissionId
-      }),
+      .notOneOf([''], 'Article title is required')
+      .nullable(),
+    cosubmission: yup
+      // validate the array as an object to allow different requirements at different positions
+      .array()
+      .of(
+        yup.object().shape({
+          title: yup.string().required('Article title is required'),
+        }),
+      )
+      .max(2, 'Cannot have more than 2 cosubmissions'),
```

Comment:
This would make sense but as the form is currently designed, this validation message would not be displayed anywhere. This is because it is validating the `cosubmission` key but we only display errors under fields and there is no field on the form with that key. This means the form would fail validation but the user would not know why (as you have already experienced).